crates/volta-core/src/tool/node/mod.rs

```diff
@@ -131,13 +131,31 @@ impl Tool for Node {
     fn install(self: Box<Self>, session: &mut Session) -> Fallible<()> {
         let node_version = self.ensure_fetched(session)?;
 
-        session.toolchain_mut()?.set_active_node(&self.version)?;
-
-        info_installed(node_version);
+        let default_toolchain = session.toolchain_mut()?;
+        default_toolchain.set_active_node(&self.version)?;
+
+        // If the user has a default version of `npm`, we shouldn't show the "(with npm@X.Y.ZZZ)" text in the success message
+        // Instead we should check if the bundled version is higher than the default and inform the user
+        // Note: The previous line ensures that there will be a default platform
+        if let Some(default_npm) = &default_toolchain.platform().unwrap().npm {
+            info_installed(self); // success: installed and set node@version as default
+
+            if node_version.npm > *default_npm {
+                info!("{} this version of Node includes {}, which is higher than your default version ({}).
+      To use the version included with Node, run `volta install npm@bundled`",
+                    note_prefix(),
+                    tool_version("npm", node_version.npm),
+                    default_npm.to_string()
+                );
+            }
+        } else {
+            info_installed(node_version); // success: installed and set node@version (with npm@version) as default
```

Comment:
I suggest dropping this comment (lovely though it is!) b/c it's apt to get out of sync with implementation.

```suggestion
            info_installed(node_version); // includes node and npm version
```